Multisig/Features/Security/SecurityCenter.swift

```diff
@@ -94,8 +94,65 @@ class SecurityCenter {
         }
     }
 
-    private static func migrateFromKeychainStorageToSecurityEnclave() {
-        //TODO:
+    func migrateFromKeychainStorageToSecurityEnclave() {
+        let task = { [unowned self] passcode in
+            do {
+                let keys = try KeyInfo.all()
+                // Migrate private keys
+                keys.compactMap { try? $0.privateKey() }.forEach {
+                    SecurityCenter.shared.import(id: DataID(id: $0.id), data: $0.keyData) { result in
+                        // TODO: Rollback on error
```

Comment:
rollback everything or just this particular input?

Reply:
Don't know yet